notebooks/InstaGeo_Demo.ipynb

```diff
@@ -1,520 +1,518 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c64fee29-dca3-4611-a43a-4412a6033751",
+   "metadata": {
+    "id": "c64fee29-dca3-4611-a43a-4412a6033751"
+   },
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import os\n",
+    "import pandas as pd\n",
+    "import numpy as np\n",
+    "from pathlib import Path"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3b11cb3a-1940-4056-a6a7-33baa375bddb",
+   "metadata": {
+    "id": "3b11cb3a-1940-4056-a6a7-33baa375bddb"
+   },
+   "outputs": [],
+   "source": [
+    "root_dir = Path.cwd()\n",
+    "chips_orig = os.listdir(os.path.join(root_dir, \"demo/chips\"))\n",
+    "chips = [chip.replace(\"chip\", \"chips/chip\") for chip in chips_orig]\n",
+    "seg_maps = [chip.replace(\"chip\", \"seg_maps/seg_map\") for chip in chips_orig]\n",
+    "\n",
+    "df = pd.DataFrame({\"Input\": chips, \"Label\": seg_maps})\n",
+    "df.to_csv(os.path.join(root_dir, \"demo/demo_train.csv\"))\n",
+    "df.to_csv(os.path.join(root_dir, \"demo/demo_val.csv\"))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6778e394-43ef-42ee-ae24-e00122e86d10",
+   "metadata": {},
+   "source": [
+    "Launch training"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "47dc76c1-26b9-47b9-8066-5c1df499b40f",
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/"
+    },
+    "collapsed": true,
+    "id": "47dc76c1-26b9-47b9-8066-5c1df499b40f",
+    "jupyter": {
+     "outputs_hidden": true
+    },
+    "outputId": "72e8f6e4-0263-431a-c313-47df5b0cbded",
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# %%bash\n",
+    "!python -m instageo.model.run --config-name=locust \\\n",
+    "    root_dir='demo' \\\n",
+    "    train.batch_size=8 \\\n",
+    "    train.num_epochs=3 \\\n",
+    "    train_filepath=\"demo/demo_train.csv\" \\\n",
+    "    valid_filepath=\"demo/demo_val.csv\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a8a98a64-0962-40e5-a52e-1e8687198862",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "Run Model Evaluation"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f8230588-2842-460c-b2e7-4f2f4175dc3e",
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/"
+    },
+    "collapsed": true,
+    "id": "f8230588-2842-460c-b2e7-4f2f4175dc3e",
+    "jupyter": {
+     "outputs_hidden": true
+    },
+    "outputId": "94515cc4-972a-4e15-d3fe-1dc28200f848",
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# %%bash\n",
+    "!python -m instageo.model.run --config-name=locust \\\n",
+    "    root_dir='demo' \\\n",
+    "    test_filepath=\"demo/demo_val.csv\" \\\n",
+    "    train.batch_size=8 \\\n",
+    "    checkpoint_path='outputs/2024-09-12/12-17-46/instageo_epoch-02-val_iou-0.00.ckpt' \\\n",
```

Comment:
Can you please check the checkpoint path , also I think we might leave it as a place holder

Reply:
Noted. I have added a placeholder as suggested